scripts/retrieveSitemap

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-curl --insecure "$1/sitemap.xml" 2>/dev/null | xmllint --xpath '//*[local-name()="loc"]/text()' --format - > sitemap.links
+curl --insecure "$1" 2>/dev/null | xmllint --xpath '//*[local-name()="loc"]/text()' --format - > sitemap.links
```

Comment:
I noticed previously that if you passed a site with a trailing "/" it would fail (looking for sitename.com//sitemap.xml for example). Does this change prevent that problem?